AngularJS/test/org/angular2/inspections/data/template/duplicate-attribute.html

```diff
@@ -0,0 +1,6 @@
+<my-comp
```

Comment:
Please use the full example from the PR thread. We need to ensure that all corner cases are covered.